website/addons/twofactor/views.py

```diff
@@ -20,7 +21,8 @@ def user_settings(user_addon, *args, **kwargs):
     if user_addon.verify_code(code):
         user_addon.is_confirmed = True
         user_addon.save()
-        return {'message': 'Successfully verified two-factor authentication.'}, http.OK
+        push_status_message('Successfully verified two-factor authentication!')
```

Comment:
@samanehsan, I'll change this - thanks. 
As for the css changes to getting_started.mako, originally I had inserted some inline css, so I removed that but I noticed that there was some other inline css that originally there that I could refactor out. Sorry for not specifying this in the PR